test/extended/olm/olmv1.go

```diff
@@ -419,3 +458,47 @@ func checkFeatureCapability(ctx context.Context, oc *exutil.CLI) {
 		g.Skip("Test only runs with OperatorLifecycleManagerV1 capability")
 	}
 }
+
+// verifyAPIEndpoint tests that an API endpoint is accessible and returns valid JSON
+func verifyAPIEndpoint(ctx g.SpecContext, urlPath string) {
+	var lastError error
+
+	// Retry API call a few times to handle potential connection issues
```

Comment:
I feel like we should _not_ be allowing ourselves to retry. IIUC, this would mean that as long as our endpoint worked 1/15 times, this test would pass.

Maybe instead, we should implement a load test with 100 concurrent requests and verify:
- 100% response rate with 200 status code (for valid requests)
- Some reasonable 99 percentile response time threshold check.

Reply:
Looking into writing a load testing now.

Reply:
There's also no retry in the new `Job`, it's a one and done operation after which the result is checked with a timeout (exit 0/exit 1)